### PR TITLE
memory check is done in % instead of KB

### PR DESCRIPTION
**Pull request by zerOnepal**
I have some humble changes here, to address my own [issue](https://github.com/sensu-plugins/sensu-plugins-redis/issues/11)

### CODE REVIEW

**Comment by analytically** [comment]
This breaks backwards compatibility. Would you be able to add two new switches that make the check use % instead instead of replacing the kB ones.

**Comment by zerOnepal** [comment]
@analytically sure !! 
or for  ease, we have have seperate check file like: **check-redis-memory-percent.rb** so the
What do you say ??

...my applogies didn't think of compatibility. :astonished:

**Comment by analytically** [comment]
No, best to keep it in the same file, but using options.

**Comment by analytically** [comment]
Actually, yes, put it in a separate check file.

**Comment by zerOnepal** [comment]
There is a small issue to adress regarding %, I will try push the changes asap.

P.S: @analytically how do you guys test the plugins, I am doing it with manual process with some hit-and-trial method. Any words of wisdom you want to share ?

**Comment by eheydrick** [comment]
It would be great if you could add some specs. Beyond that, do a smoke test against a real instance.

**Comment by eheydrick** [comment]
Any updates on this?

**Comment by zerOnepal** [comment]
@eheydrick kinda occupied,  I will try my best push changes this weekend

**Comment by zerOnepal** [comment]
@eheydrick /cc @analytically 
Pushed the changes on sperate file as **check-redis-memory-percentage.rb** and I have reverted the old file.

Here is snapshot results in my notification channel.
sensu_check_memory_result_

**Comment by envintus** [comment]
@eheydrick, @zerOnepal Is this pull request still under consideration to be merged?

**Comment by zerOnepal** [comment]
@envintus i have pushed the changes.. 

head-mouse
wonder what's taking too long for it to merge...

File: bin/check-redis-memory-percentage.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+#
+# Checks Redis INFO stats and limits values
+# ===
+#
+# Copyright (c) 2012, Panagiotis Papadomitsos <pj@ezgr.net>
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
```
Comment (eheydrick) [inline bin/check-redis-memory-percentage.rb]: can you add the standard header per http://sensu-plugins.io/docs/developer_guidelines.html#coding-style

**Comment by eheydrick** [comment]
@zerOnepal can you add the new plugin to the CHANGELOG and README and we'll get this merged.

File: bin/check-redis-memory-percentage.rb (hunk 1 above)
Comment (zerOnepal) [inline bin/check-redis-memory-percentage.rb]: sure... my bad

**Comment by zerOnepal** [comment]
@eheydrick done
happy_keed_playing_with_water

**Comment by eheydrick** [comment]
Thanks @zerOnepal!

**Pull request merged**